src/rest.cpp

```diff
@@ -387,7 +386,8 @@ static bool rest_tx(HTTPRequest *req, const std::string &strURIPart)
 
     CTransactionRef tx;
     uint256 hashBlock = uint256();
-    if (!GetTransaction(hash, tx, Params().GetConsensus(), hashBlock, true))
+    int64_t txTime = GetTime();
```

Comment:
nit: why initializing to `GetTime()`? in `GetTransaction` first thing done with txTime is to set it to 0.

Reply:
fixed, thx!